nncf/torch/binarization/binarize_functions.py

```diff
@@ -140,3 +150,7 @@ def backward(ctx, grad_output):
                     grad_threshold = grad_threshold.sum(idx, keepdim=True)
 
         return grad_input, grad_scale, grad_threshold
+
+    @staticmethod
+    def jvp(ctx: Any, *grad_inputs: Any) -> Any:
```

Comment:
This method became an abstract in torch.autograd.Function.
It defines a formula for differentiating the operation with forward mode automatic differentiation (https://pytorch.org/tutorials/intermediate/forward_ad_usage.html)
It works fine without.

Reply:
Eventually, it didn't help to fix pylint warnings about abstract method. Added exception for `torch.autograd.Function`